pages/instructorGradebook/instructorGradebook.ejs

```diff
@@ -2,42 +2,99 @@
 <html lang="en">
   <head>
     <%- include('../partials/head'); %>
+    <script src="<%= node_modules_asset_path('bootstrap-table/dist/bootstrap-table.min.js') %>"></script>
+    <script src="<%= node_modules_asset_path('bootstrap-table/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.js') %>"></script>
+    <link href="<%= node_modules_asset_path('bootstrap-table/dist/bootstrap-table.min.css') %>" rel="stylesheet" />
+    <link href="<%= node_modules_asset_path('bootstrap-table/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.css') %>" rel="stylesheet" />
   </head>
+  <script id="grade-data">
+    let gradeData = <%- JSON.stringify(user_scores_data) %>;
```

Comment:
I'm worried about https://sophiebits.com/2012/08/03/preventing-xss-json.html

In the file editor page we do this on the server: https://github.com/PrairieLearn/PrairieLearn/blob/f37c625f9c67f2037d57277326d2e68927b9d9d4/pages/instructorFileEditor/instructorFileEditor.js#L124

and embed it into the HTML with: https://github.com/PrairieLearn/PrairieLearn/blob/f37c625f9c67f2037d57277326d2e68927b9d9d4/pages/instructorFileEditor/instructorFileEditor.ejs#L43

and then in client-JS we unpack it: https://github.com/PrairieLearn/PrairieLearn/blob/f37c625f9c67f2037d57277326d2e68927b9d9d4/pages/instructorFileEditor/instructorFileEditorClient.js#L82

Reply:
Resolved by escaping the input strings.

Reply:
Perfect!